recipes/desdeo-vis/meta.yaml

```diff
@@ -0,0 +1,60 @@
+{% set name = "desdeo-vis" %}
+{% set version = "0.1.2" %}
+{% set file_ext = "tar.gz" %}
+{% set hash_type = "sha256" %}
+{% set hash_value = "d70a42e3aa6e4f8ef1c731b17a7baefea39f9d6d015597827fad9b9e42f27e30" %}
+
+package:
+  name: '{{ name|lower }}'
+  version: '{{ version }}'
+
+source:
+  fn: '{{ name }}-{{ version }}.{{ file_ext }}'
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.{{ file_ext }}
+  '{{ hash_type }}': '{{ hash_value }}'
+
+build:
+  noarch: python
+  number: 0
+  script: python -m pip install --no-deps --ignore-installed .
+
+requirements:
+  build:
+    - pip
+    - python >= 3.6
```

Comment:
So should have caught this, but I missed it. There shouldn't be a space after `>=`.

Reply:
Fixed in PR ( https://github.com/conda-forge/staged-recipes/pull/6311 ).